### ISSUE TITLE
how the weeks name changes Example: mommy must murderer

### ISSUE THREAD

**Issue by Gabriel66665** [luckydog7/Funkin-android#448]

**Comment by bloodheroC** [comment]
Its impossible you need to add week to changes the name

**Comment by Rs232k** [comment]
What you have? Phone or pc? Only with pc (you can theoretically edit libapplicationmain.so file, but it maybe very hard.)

**Comment by bloodheroC** [comment]
I have a phone

**Comment by Gabriel66665** [comment]
I have a phone

**Comment by Gabriel66665** [comment]
I have a phone too

**Comment by ThatGuyW** [comment]
ngl you can change it on mobile but it's a code file

**Comment by Gabriel66665** [comment]
What is the name of the code?

**Comment by DabKnight** [comment]
I dont think this is gonna work but a few days ago when i asked how to make a custom week someome said to edit a file named "StoryMenuState.hx"
Its in the source code.

**Comment by Gabriel66665** [comment]
How strange I didn't find anything with that name: / very strange do you know how to tell me or ask someone who knows where the location of this file is?

**Comment by DabKnight** [comment]
This is what they said:
If you can, go into the code. Open StoryMenuState.hx and add a dash behind week 6. After you add the dash, make a new line and use [] to make a new week statement (for example: ['Improbable-Outset']) then go to the if statement with the 7 true statements, add another dash and add another true. then go to the character tab and type in your character names using [] again (example: ['dad', 'bf', 'gf']) and go to the line that has the week names and add another dash behind week 6. After, go to a new line and type (this time in "") your week name (example: "CIRCUS CLOWN") and then you have a week added

**Comment by Gabriel66665** [comment]
but I'm asking where is the file to change the names of the weeks because I don't want to make a custom one but to change the name of the original weeks

**Comment by Rs232k** [comment]
> but I'm asking where is the file to change the names of the weeks because I don't want to make a custom one but to change the name of the original weeks

You have a phone, you can't

**Comment by WyxosYT** [comment]
> but I'm asking where is the file to change the names of the weeks because I don't want to make a custom one but to change the name of the original weeks

It's on the "StoryMenuState.hx" file![Screenshot_20210422_023305_com.rhmsoft.code.jpg](https://user-images.githubusercontent.com/82682892/115643313-730f7b80-a314-11eb-996c-df45853f76fd.jpg)

**Comment by Gabriel66665** [comment]
Wait more I download the source code zip?

**Comment by AGuyLucky** [comment]
I have the source code modified but i dont know how build the game
